doc/operation-and-maintenance/Reloading-configuration-on-a-running-system.md

```diff
@@ -22,3 +27,147 @@ The following options' changes will be ignored when using `mongooseimctl` tool:
 * s2s_*
 * all_metrics_are_global
 * odbc_*
+
+
+### Node-specific options
+
+Very rarely we want different configs for each node in cluster.
```

Comment:
Very rarely do we want different configs for each node in a cluster.

Reply:
wow, why? :D